lib/restclient/request.rb

```diff
@@ -250,15 +254,19 @@ def process_url_params url, headers
     def make_headers user_headers
       unless @cookies.empty?
 
-        # Validate that the cookie names and values look sane. If you really
-        # want to pass scary characters, just set the Cookie header directly.
```

Comment:
In my reading of the code, I'm not sure what "just set the Cookie header directly" refers to.  Line 129 seems to actually prevent this:

```ruby
@cookies = @headers.delete(:cookies) || args[:cookies] || {}
```